tests/common/gllayerrenderer.cpp

```diff
@@ -54,6 +54,7 @@ bool GLLayerRenderer::Init(uint32_t width, uint32_t height, uint32_t format,
     return false;
   }
 
+#ifndef DEBUG_USE_GL_TEXTURE2D
```

Comment:
Do we need to use MACRO here?
Or we can just use a differentialzed initial function in inherited class